src/app.ts

```diff
@@ -13,14 +14,51 @@ interface TemperatureSummary {
   average: number
 }
 
+const summaryTemperature: { [item: string]: { [day: string]: number[] } } = {}
+
 export function processReadings(readings: TemperatureReading[]): void {
-  // add here your code
+  for (const read of readings) {
+    if (read.city in summaryTemperature) {
+      if (read.time.toDateString() in summaryTemperature[read.city]) {
+        summaryTemperature[read.city][read.time.toDateString()].push(
+          read.temperature,
+        )
+      } else {
+        summaryTemperature[read.city][read.time.toDateString()] = [
+          read.temperature,
+        ]
+      }
+    } else {
+      summaryTemperature[read.city] = {
+        [read.time.toDateString()]: [read.temperature],
+      }
+    }
+  }
 }
 
 export function getTemperatureSummary(
   date: Date,
   city: string,
 ): TemperatureSummary | null {
-  //add here your code
+  if (
+    city in summaryTemperature &&
+    date.toDateString() in summaryTemperature[city]
+  ) {
+    let values = summaryTemperature[city][date.toDateString()]
```

Comment:
"high" and "low" values are the highest and lowest temperatures in a specific day(). Also "average" is the average of the day.